aiida/cmdline/commands/cmd_computer.py

```diff
@@ -556,6 +572,7 @@ def computer_configure():
 def computer_config_show(computer, user, defaults, as_option_string):
     """Show the current configuration for a computer."""
     from aiida.common.escaping import escape_for_bash
+    from aiida.transports import cli as transport_cli
```

Comment:
just from the PR diff it looks like this is being introduced without being used (weird - shouldn't pylint catch this?)

Reply:
no its just been moved from the top of the file to inside the function, so that its lazy loaded. Its still used inside that function